example/extensions/lib_custom_op/README.md

```diff
@@ -67,76 +67,85 @@ mx.library.load(‘libmyop_lib.so’)
 mx.nd.my_op(...)
 ```
 
-### Writing Regular Custom Operator:
+### Writing Regular Custom Operator
 
 There are several essential building blocks for making a (stateless) custom operator:
```

Comment:
lets not mention "stateless", lets just talk about custom operators and how to make an operator stateful